src/main/java/org/llorllale/cactoos/matchers/HasValues.java

```diff
@@ -67,19 +58,11 @@ public HasValues(final X... expected) {
      * @param expected The expected values within unit test.
      */
     public HasValues(final Iterable<X> expected) {
-        super();
-        this.expected = expected;
-    }
-
-    @Override
-    public void describeTo(final Description dsc) {
-        dsc.appendValue(new TextOf(this.expected));
-    }
-
-    @Override
-    protected boolean matchesSafely(final Iterable<X> actual,
-        final Description dsc) {
-        dsc.appendValue(new TextOf(actual));
-        return new ListOf<>(actual).containsAll(new ListOf<>(this.expected));
+        super(
+        // @checkstyle IndentationCheck (3 line)
+        actual -> new ListOf<>(actual).containsAll(new ListOf<>(expected)),
+        desc -> desc.appendText("contains ").appendValue(new TextOf(expected)),
+        (actual, desc) -> desc.appendValue(new TextOf(actual))
+        );
```

Comment:
@victornoel is there a particular reason not to respect standard indentation rules here?

Reply:
@scristalli yes, there is a bug in qulice (https://github.com/teamed/qulice/issues/898), if I put one indentation for the arguments, I get:
```
Checkstyle: src/main/java/org/llorllale/cactoos/matchers/HasValues.java[62]: 'lambda arguments' have incorrect indentation level 12, expected level should be 16. (IndentationCheck)
```

If I follow the recommendation, I get:
```
Checkstyle: src/main/java/org/llorllale/cactoos/matchers/HasValues.java[62]: 'lambda arguments' have incorrect indentation level 16, expected level should be 20. (IndentationCheck)
Checkstyle: src/main/java/org/llorllale/cactoos/matchers/HasValues.java[62]: Indentation (16) must be same or less than previous line (8), or bigger by exactly 4 (CascadeIndentationCheck)
```

That being said, I will at least indent if once for readability.